presentation/src/main/java/com/anytypeio/anytype/presentation/relations/option/CreateOrEditOptionViewModel.kt

```diff
@@ -152,6 +150,30 @@ class CreateOrEditOptionViewModel(
         )
     }
 
+    private fun updatedValues(
+        newOptionId: Id,
+        relation: ObjectWrapper.Relation,
+        obj: Struct
+    ): List<Id> {
+        return when (relation.format) {
+            Relation.Format.STATUS -> {
+                listOf(newOptionId)
+            }
+            Relation.Format.TAG -> {
+                val result = mutableListOf<Id>()
```

Comment:
return buldList {  
val value = obj[relation.key]
                if (value is List<*>) {
                    addAll(value.typeOf())
                } else if (value is Id) {
                    add(value)
                }
                add(newOptionId)

  }

Reply:
done